src/sous_chef/templates/base.html

```diff
@@ -70,6 +70,12 @@
     {% endif %}
     <!-- WARNING :  Do not move this line before a manual import of script! -->
     {% block extrajs %}{% endblock %}
+
+    <script type="application/javascript">
+      $(function () {
+        for(var f=document.forms,i=f.length;i--;)f[i].setAttribute("novalidate",i)
+      });  // disable HTML5 form validation
+    </script>
```

Comment:
@erozqba  That makes sense. I'll look for a way of working with hidden fields then...